docs/bankid.md

```diff
@@ -420,6 +418,8 @@ These are only necessary if you plan to store your certificates in Azure KeyVaul
 {
     "ActiveLogin:BankId:ClientCertificate": {
         "UseManagedIdentity": true,
+        "ManagedIdentityType": ManagedIdentityType.SystemAssigned,
```

Comment:
Enums in JSON is written in quotes, without the type I think. See for example:
https://docs.microsoft.com/en-us/answers/questions/643286/appsettingsjson-with-enum.html